common/lib/common-utils/src/bufferNode.ts

```diff
@@ -61,3 +61,14 @@ export function stringToBuffer(input: string, encoding: string): ArrayBufferLike
  */
 export const bufferToString = (blob: ArrayBufferLike, encoding: string): string =>
     IsoBuffer.from(blob).toString(encoding);
+
+/**
+ * Convert Uint8Array array to ArrayBuffer
+ * @param array - array to convert to ArrayBuffer
+ */
+export function Uint8ArrayToArrayBuffer(array: Uint8Array): ArrayBuffer {
```

Comment:
Given that impl is the same between node & browser, can you please share this code and have one copy?

Reply:
my bad for asking you to duplicate it; i thought it would be different for some reason 🙃